galleries/tutorials/artists.py

```diff
@@ -11,15 +11,15 @@
 
 There are three layers to the Matplotlib API.
 
-* the :class:`matplotlib.backend_bases.FigureCanvas` is the area onto which
+* the :class:`!matplotlib.backend_bases.FigureCanvas` is the area onto which
```

Comment:
Actually, the previous way of expression is not exactly great - there is just no class
`matplotlib.backend_bases.FigureCanvas`. We have a logical concept of *FigureCanvas* that's realized through the subclasses of `FigureCanvasBase`. Maybe

```suggestion
* the *FigureCanvas* (subclasses of `.FigureCanvasBase`) is the area onto which
```

And subsequently only use FigureCanvas as a given name (for the logical concept) without further highlighting.

Or better, switch to ``:term:`FigureCanvas` ``, but that'd require first setting up a gloassary #28889.

But we can also leave all this for later.